### DISCUSSION TITLE
Bot high ping

### DISCUSSION THREAD

**Discussion by Ozak93** [jagrosh/MusicBot#1385] [Troubleshooting]
Hello, First i want to thank you for the bot, it's great! i've tried several self-host bots but all had issues except yours sofar!

I noticed the bot has very high ping! it's running in a Contabo VPS in Germany and connected to a server in Rotterdam but it's ping is above 200! 
this is the outpot of /ping 
Ping: 220ms | Websocket: 121ms 

what can i do to reduce the ping?

**Comment by MineTurtlee** [comment]
You can see if the bot is from a really far region and you start the vc that is right near your country. Imagine that, and that's ping!
I've experienced this before so no doubt!

**Comment by AnimMouse** [comment]
https://github.com/jagrosh/MusicBot/discussions/1104#discussioncomment-2028992
It's not voice channel ping. It's text channel ping & ping to Discord's WebSocket at us-east.
As long as your bot is hosted near your voice channel's region, you are good to go.
